packages/calypso-build/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
+
+- Add `overrides` option to `transpile`, to be used in `babel-loader`.
+  This allows for providing e.g. extra plugin configuration, beyond what's in the `configFile`.
```

Comment:
I think it's this particular line that had my guard up :sweat_smile: I wouldn't really want to expose this to `calypso-build` users as a recommended way to customize behavior.

The use case you're describing here seems quite specific (compile-time evaluation of app config flags), and rather tied to the way we implement it; I wouldn't want people to jump to the conclusion that this is a standard way they should use to configure `calypso-build`. (Note that none of the config options in `calypso-build/webpack/*.js` are documented in `calypso-build`'s README.)

I realize we need _something_ in the Changelog. Maybe let's just remove this line? Fine to keep the one above.